addons/hr_timesheet/views/hr_timesheet_portal_templates.xml

```diff
@@ -179,13 +179,23 @@
                         <tr>
                             <div t-if="task.subtask_effective_hours">
                                 <t t-if="is_uom_day">
-                                    <td><a t-att-href="'/my/timesheets?search_in=parent_task_id&amp;search=%s' % task.id"><strong>Days recorded on sub-tasks: </strong></a></td>
+                                    <t t-if="enable_sub_task_link">
+                                        <td><a t-att-href="'/my/timesheets?search_in=parent_task_id&amp;search=%s' % task.id"><strong>Days recorded on sub-tasks: </strong></a></td>
+                                    </t>
+                                    <t t-else="">
+                                        <td><strong>Days recorded on sub-tasks: </strong></td>
+                                    </t>
```

Comment:
We could do the changes without changing the current structure. 🙂 

Something like:

```suggestion
                                    <td>
                                        <a t-if="enable_sub_task_link" t-att-href="'/my/timesheets?search_in=parent_task_id&amp;search=%s' % task.id"><strong>Days recorded on sub-tasks: </strong></a>
                                        <strong t-else="">Days recorded on sub-tasks: </strong>
                                    </td>
```

ps: it is important in stable versions to avoid altering the XML structure to avoid breaking xpath made inside eventual custom modules. 🙂